### PR TITLE
chore: upgrade @wdio and specs

### PR DESCRIPTION
**Pull request by 9renpoto**
🙏

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,26 +71,30 @@
...
   "size-limit": [
     {
       "path": "lib/index.js",
       "limit": "300 B"
     }
   ],
-  "types": "lib/index.d.ts"
+  "types": "lib/index.d.ts",
+  "dependencies": {
+    "@types/mocha": "^7.0.2"
```
Comment (9renpoto) [inline package.json]: 😰

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,13 @@
-MIT License
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```
Comment (shoota) [inline LICENSE:1]: Nice suggestion.
Ok..., we can change to this, but I hope keep MIT...

File: .github/size-plugin.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+base-branches:
```
Comment (9renpoto) [inline .github/size-plugin.yml:1]: https://probot.github.io/apps/size-plugin/

**Review by shoota — COMMENTED** [review]
TOO LARGE and mixed context.
Please separate PR each problems.

**Comment by 9renpoto** [comment]
Its too difficult for  me, because failed master branch 😰

**Pull request closed without merging**